api-report/tree2.api.md

```diff
@@ -1863,12 +1863,30 @@ export interface Sequence2<TTypes extends AllowedTypes> extends TreeField {
     readonly asArray: readonly UnboxNodeUnion<TTypes>[];
```

Comment:
Was 'slice()' an accidental omission?  (Once we have 'slice()', I believe 'asArray' is unnecessary.)

Reply:
Good question. It was intentional - see the table halfway down [this document](https://microsoft.sharepoint.com/:fl:/s/118fd7a1-4341-47ce-a4eb-c83ed61e0317/EfheP4kqenNNv33V6isSZoQBSvWn4xKuD2ZcZXvR-QiBhA?e=6hDmiC&nav=cz0lMkZzaXRlcyUyRjExOGZkN2ExLTQzNDEtNDdjZS1hNGViLWM4M2VkNjFlMDMxNyZkPWIlMjFYaFJYWnVOMDVVNmJlUUNvTmxDMWM4eXpiNW9USkdWRnQyVnI5Q0l3b014N2NXODZDVURXVHAyYy1GQWJSTXhtJmY9MDFDWkRDU043WUxZN1lTS1QyT05HMzY3T1Y1SVZSRVpVRSZjPSUyRiZhPUxvb3BBcHAmcD0lNDBmbHVpZHglMkZsb29wLXBhZ2UtY29udGFpbmVyJng9JTdCJTIydyUyMiUzQSUyMlQwUlRVSHh0YVdOeWIzTnZablF1YzJoaGNtVndiMmx1ZEM1amIyMThZaUZZYUZKWVduVk9NRFZWTm1KbFVVTnZUbXhETVdNNGVYcGlOVzlVU2tkV1JuUXlWbkk1UTBsM2IwMTROMk5YT0RaRFZVUlhWSEF5WXkxR1FXSlNUWGh0ZkRBeFExcEVRMU5PV1VSTFNqWkZSVFZJTWxwYVJVazBURVJRU0RJeVJUSkNValklM0QlMjIlMkMlMjJpJTIyJTNBJTIyY2NjYjdlZDktZGJkYy00NWZkLTkxOGQtNjU0NGYzMTFkYTg5JTIyJTdE). I'm not sure if I still agree with the decision or not. Seen as a strictly more general version of `asArray`, and not having any other semantic connotations, I agree it would supersede `asArray`. However, `asArray` might be a more discoverable way to get readonly array methods that we don't put on sequence directly. Also, I believe @CraigMacomber has particular reasons why we want `asArray` specifically as a getter property and not a method.